### DISCUSSION TITLE
lat/lon format?

### DISCUSSION THREAD

**Discussion by keshka** [rocksdanister/weather#56] [Q&A]
How does Lively Weather prefer latitude / longitude date formatted? 
I tried this: 45.56467°N 117.92617°W
and pressed enter, LW just blinks and empties the box.

**Comment by rocksdanister** [comment]
`Latitude, Longitude`, there is a demo clip in the release:

https://github.com/rocksdanister/weather/releases/tag/v1.0.28.0

Sometimes it may not find anything for the given coordinates so nothing happens.. better to just search by name.

**Comment by keshka** [comment]
Understood, however I live in a rural area and the nearest town has a completely different weather pattern. This is the main reason I like NOAA. I can get a pinpoint forecast that is 95% accurate. I looked at NOAA's website for information on an API key. The have it for free but I don't understand how to implement it nor Microsoft Visual Basic as I am an old school C programmer. 

I love the set of Lively apps, thank you so much for creating them.

**Reply by rocksdanister** [reply]
You can try the openweathermap webpage to see if the location is available (zip code search etc..)

Lat, Long search is not available with Open-meteo.

I don't have a timeframe for new weather provider, already burned out from implementing openweathermap.. will see.

**Reply by keshka** [reply]
No worries Dani! 
I am currently using openweathermap and it's doing well. It latched in on a town about 15 miles away when given coordinates and the forecast is pretty close to my area.
Thank you again!
Keshka
